docs/self-managed/setup/deploy/local/docker-compose.md

```diff
@@ -1,26 +1,254 @@
+- **Docker:** Version 20.10.16+
+- **Keycloak:** (Local development only). Add Keycloak to resolve to 127.0.0.1 on your local machine, and set `KEYCLOAK_HOST=keycloak` in the `.env` file for token refresh and logout functionality.
+
+## Run Camunda 8 with Docker Compose
+
+To start a complete instance of Camunda 8 Self-Managed environment locally:
+
+1. Clone the [Camunda Self-Managed repository](https://github.com/camunda/camunda-self-managed), and navigate to the directory of the desired Camunda version (for example, `/version/camunda-8.6`).
+2. Run the following command:
+
+```shell
+docker compose up -d
+```
+
+1. Wait for the environment to initialize. This may take several minutes. Monitor the logs, especially the Keycloak container log, to ensure the components have started.
+
+### Configuration options
+
+Running `docker compose up -d` starts all Camunda components, including Identity. The [Camunda Self-Managed repository](https://github.com/camunda/camunda-self-managed) also contains the additional configuration files for lightweight development.
+
+- **docker-compose.yaml:** Contains all Camunda 8 Components for a full stack deployment: Zeebe, Operate, Tasklist, Connectors, Optimize, Identity, Elasticsearch, Keycloak, Web Modeler, and PostgreSQL.
+- **docker-compose-core.yaml:** Contains Camunda 8 Orchestration cluster components: Zeebe, Tasklist, Operate, and Connectors.
+- **docker-compose-web-modeler.yaml:** Contains the Camunda 8 Web Modeler standalone installation. For more information, see the [Web Modeler instructions](#web-modeler).
+
+To start Camunda with an alternate configuration, specify a file using the following command:
+
+```shell
+docker compose -f docker-compose-core.yaml up -d
+```
+
+### Access components
+
+Running components can be accessed with the username `demo` and password `demo`:
+
+- Operate: [http://localhost:8081](http://localhost:8081)
+- Tasklist: [http://localhost:8082](http://localhost:8082)
+- Optimize: [http://localhost:8083](http://localhost:8083)
+- Identity: [http://localhost:8084](http://localhost:8084)
+- Web Modeler [http://localhost:8070](http://localhost:8070)
+- Elasticsearch: [http://localhost:9200](http://localhost:9200)
+
+Keycloak is used to manage users, and can be accessed with the user `admin` and password `admin`:
+
+- Keycloak: [http://localhost:18080/auth/](http://localhost:18080/auth/)
+
+The workflow engine Zeebe is available using gRPC:
+
+- Zeebe: `localhost:26500`
+
+### Stop Camunda 8
+
+Run the following command to stop Camunda 8:
+
+```shell
+docker compose down -v
+```
+
+:::caution
+This will also delete any data you created.
+:::
+
+To tear down the environment and keep any data, run the following command:
+
+```shell
+docker compose down
+```
+
+Zeebe, Operate, Tasklist, along with Optimize require a separate network from Identity as you'll see in the docker-compose file.
```

Comment:
I wasn't sure about the necessity of this line, or if it needs to be in this section - I think we can remove it, but wanted to point it out.

Reply:
I've removed it :+1: